primitives/core/src/lib.rs

```diff
@@ -48,11 +48,11 @@ pub use sp_debug_derive::RuntimeDebug;
 #[cfg(feature = "std")]
 pub use impl_serde::serialize as bytes;
 
-#[cfg(feature = "full_crypto")]
 pub mod hashing;
```

Comment:
We can't include this crate in the runtime (`full-crypto` isn't set for runtime builds) because it has some C dependencies. You should call a host function for hashing just like it was done in `beefy-mmr`:
```rust
sp_io::hashing::keccak_256()
```